openapi/ga4gh-tool-discovery.yaml

```diff
@@ -124,6 +124,10 @@ paths:
           description: |-
             OPTIONAL for tool registries that support aliases.
             If provided will only return entries with the given alias.
+        - name: toolClass
+          type: string
+          in: query
+          description: 'Filter tools by the name of the subclass (#/definitions/ToolClass)'
```

Comment:
The descriptions for the other optional parameters start with the word `OPTIONAL`. I actually don't like that, and would prefer to rely on the lack of `required: true` alone to indicate optional, but I think you should be consistent one way or another.

Reply:
The big OPTIONAL is actually for features that we deemed are optional parts of the standard as opposed to optional parameters. I agree that it can be confusing so will re-word